### PR TITLE
We have moved past the need to have a Homestuck Reference

### PR DESCRIPTION
**Pull request by MassacreDemigirl**
## About The Pull Request
This PR makes everyone a favor by removing the homestuck reference SORD from the game.

## Why It's Good For The Game
We do not need a shitty reference of a weapon

## Changelog

:cl:
del: Deletes the shitty reference that is SORD.
/:cl:

### CODE REVIEW

**Comment by Colovorat** [comment]
soul removal

**Comment by necromanceranne** [comment]
This thing is so unspeakably shitty we're are having a hard time even keeping it in the codebase.

**Comment by EricZilla** [comment]
Removal without a better topical replacement shitty weapon smh

**Comment by Kylerace** [comment]
i dont know what homestuck is and i like sord on its own go away

**Comment by cacogen** [comment]
I stuck with Homestuck long enough to understand this reference and although I would never pick this weapon and I never play chaplain I do not want to see it removed, strictly for nostalgia purposes. There's also no justification for removing it other than Homestuck still carrying a stigma somehow. It was funny and clever. Get over it, bigot.

File: code/game/objects/items/gift.dm [hunk 1]
```diff
@@ -54,7 +54,7 @@ GLOBAL_LIST_EMPTY(possible_gifts)
 		M.visible_message(span_danger("Oh no! The present that [M] opened had nothing inside it!"))
 
 /obj/item/a_gift/proc/get_gift_type()
-	var/gift_type_list = list(/obj/item/sord,
+	var/gift_type_list = list(
```
Comment (Thunder12345) [inline code/game/objects/items/gift.dm:57]: var/gift_type_list = list(/obj/item/storage/wallet,

**Comment by Thunder12345** [comment]
You also need to remove the sord from _maps/RandomZLevels/caves.dmm

**Comment by carshalash** [comment]
You're forgetting the homestuck bible too.

**Comment by ErdinyoBarboza** [comment]
which bibble is that one?

**Pull request closed without merging**